packages/playground/php-cors-proxy/cors-proxy.php

```diff
@@ -158,8 +159,11 @@
     http_response_code(502);
     echo "Bad Gateway – curl_exec error: " . curl_error($ch);
 } else {
-    $httpCode = curl_getinfo($ch, CURLINFO_HTTP_CODE);
-    @http_response_code($httpCode);
+    if (!$httpcode_sent) {
```

Comment:
Since we may have already sent the HTTP response code and set `$httpcode_sent`, let's avoid setting the response code again.